solo-chains/runtime/dancelight/src/lib.rs

```diff
@@ -209,6 +210,85 @@ pub fn native_version() -> NativeVersion {
     }
 }
 
+/// Aggregate message origin for the `MessageQueue` pallet.
+///
+/// Can be extended to serve further use-cases besides just UMP. Is stored in storage, so any change
+/// to existing values will require a migration.
+#[derive(Encode, Decode, Clone, MaxEncodedLen, Eq, PartialEq, RuntimeDebug, TypeInfo)]
+pub enum AggregateMessageOrigin {
+    /// Inbound upward message.
+    #[codec(index = 0)]
+    Ump(UmpQueueId),
```

Comment:
Also, you might not need this as well. Since it looks like the only use for outbound queue pallet for `AggregateMessageOrigin` is to enqueue it for later processing by itself in which case the origin is ignored anyway. 

If this requires widespread changes, ignore this comment!

Reply:
I believe this is for backwards compatibility, before this PR we used the `AggregateMessageOrigin` from the paras_inclusion pallet, which has Ump as index 0. Also we probably will need to support this at some point in the future (for xcm maybe?). So I'm ignoring this

Reply:
*disregard*